hw/dv/tools/vcs/vcs.tcl

```diff
@@ -2,25 +2,20 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
-# TCL file invoked from VCS's simv at run-time using this: -ucli -do <this file>
-
-# Get some environment variables we need.
-set en_waves 0
+# TCL file invoked from VCS simulations at run-time using this: -ucli -do <this file>
 set tool_srcs_dir ""
-if {[info exists ::env(EN_WAVES)]} {
-    set en_waves "$::env(EN_WAVES)"
-}
 if {[info exists ::env(TOOL_SRCS_DIR)]} {
-    set tool_srcs_dir "$::env(TOOL_SRCS_DIR)"
+  set tool_srcs_dir "$::env(TOOL_SRCS_DIR)"
 } else {
-    puts "ERROR: tool script run without TOOL_SRCS_DIR environment variable."
-    quit
+  puts "ERROR: tool script run without TOOL_SRCS_DIR environment variable."
+  quit
 }
 
-# If wave dumping is enabled, run waves.tcl
-if {"$en_waves" == 1} {
-    source "${tool_srcs_dir}/waves.tcl"
-}
+source "${tool_srcs_dir}/common.tcl"
+source "${tool_srcs_dir}/waves.tcl"
 
-run
-quit
+setDefault skip_run 0
+if {$skip_run == 0} {
+  run
+  quit
+}
```

Comment:
this file looks identical to xcelium.tcl. Should we just have one or even consolidate this in the wave.tcl?

Reply:
I see. Probably it's fine.

Reply:
I actually consolidated it to one common tcl file sourced by both.